CacheService/docker_context/LegacyStockServerHandler.py

```diff
@@ -17,6 +25,28 @@ async def quoteSocket(self):
         return reader, writer
 
     async def getQuote(self, trans_num, user_id, stock_symbol):
+        if stock_symbol in self.tracker.keys():
+            runningTask = self.tracker[stock_symbol]
```

Comment:
shouldn't it get removed when it does finish running

Reply:
self.tracker[stock_symbol] is removed at the end of __getQuoteFunc